tests/psoc6/i2c_hard.py

```diff
@@ -0,0 +1,9 @@
+###
+import machine
```

Comment:
Not required to import the whole of `machine` module. Handled by the next import statement

Reply:
Right, no longer needed